src/App.tsx

```diff
@@ -51,6 +51,24 @@ export function App() {
     }
   }
 
+  const onCheckpoint = () => {
+    mutation.tempCheckpoint1 = Date.now() - mutation.start
+    if (mutation.checkpoint1 == 0) {
+      mutation.checkpoint1 = mutation.tempCheckpoint1
+      return
+    }
+
+    mutation.checkpointDifference = mutation.tempCheckpoint1 - mutation.checkpoint1
+    if (mutation.checkpointDifference < 0) {
+      mutation.checkpoint1 = mutation.tempCheckpoint1
+    }
+
+    set({
```

Comment:
set({ showCheckpoint: true })

three lines dude 😅

Reply:
oof, i assumed prettier would handle that 😢